hosts

```diff
@@ -4,6 +4,10 @@ ny1.hashbang.sh
 sf1.hashbang.sh
 to1.hashbang.sh
 
+[shell_servers]
+ansible_user=root
```

Comment:
Eeeh, experience (#! and otherwise) says that having a method of getting root without relying on LDAP is good  :3